src/patternfly/components/Masthead/masthead.scss

```diff
@@ -0,0 +1,234 @@
+  --pf-c-masthead--inset: var(--pf-global--spacer--lg);
+
+  // insets
+  --pf-c-masthead--m-page-insets--inset: var(--pf-global--spacer--md);
+  --pf-c-masthead--m-page-insets--xl--inset: var(--pf-global--spacer--lg);
+
+  // stack
+  --pf-c-masthead--m-display-stack--FlexWrap: wrap;
+  --pf-c-masthead--m-display-stack__main--Order: -1;
+  --pf-c-masthead--m-display-stack__main--FlexBasis: 100%;
+  --pf-c-masthead--m-display-stack__main--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__main--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__main--MarginRight: 0;
+  --pf-c-masthead--m-display-stack__main--before--BorderBottom: var(--pf-c-masthead__main--before--BorderBottomWidth) solid var(--pf-c-masthead__main--before--BorderBottomColor);
+  --pf-c-masthead--m-display-stack__content--Order: 1;
+  --pf-c-masthead--m-display-stack__content--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__content--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__content--MarginLeft: 0;
+  --pf-c-masthead--m-display-stack__content--c-nav--m-horizontal--MarginRight: calc(var(--pf-c-masthead--inset) * -1);
+  --pf-c-masthead--m-display-stack__content--c-nav--m-horizontal--MarginLeft: calc(var(--pf-c-masthead--inset) * -1);
+
+  // inline
+  --pf-c-masthead--m-display-inline--FlexWrap: nowrap;
+  --pf-c-masthead--m-display-inline__main--Order: 0;
+  --pf-c-masthead--m-display-inline__main--FlexBasis: auto;
+  --pf-c-masthead--m-display-inline__main--PaddingTop: 0;
+  --pf-c-masthead--m-display-inline__main--PaddingBottom: 0;
+  --pf-c-masthead--m-display-inline__main--MarginRight: calc(var(--pf-global--gutter--md) / 2); // split gutter
+  --pf-c-masthead--m-display-inline__main--before--BorderBottom: 0;
+  --pf-c-masthead--m-display-inline__content--Order: 0;
+  --pf-c-masthead--m-display-inline__content--PaddingTop: 0;
+  --pf-c-masthead--m-display-inline__content--PaddingBottom: 0;
+  --pf-c-masthead--m-display-inline__content--MarginLeft: calc(var(--pf-global--gutter--md) / 2); // split gutter
+  --pf-c-masthead--m-display-inline__content--c-nav--m-horizontal--MarginRight: 0;
+  --pf-c-masthead--m-display-inline__content--c-nav--m-horizontal--MarginLeft: 0;
+
+  // main border
+  --pf-c-masthead__main--before--Right: calc(var(--pf-c-masthead--inset) * -1);
+  --pf-c-masthead__main--before--Left: calc(var(--pf-c-masthead--inset) * -1);
+  --pf-c-masthead__main--before--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-masthead__main--before--BorderBottomColor: var(--pf-global--palette--black-600);
+
+  // toggle
+  --pf-c-masthead__toggle--MarginRight: calc(var(--pf-global--gutter--md) / 2);
+
+  // light 100
+  --pf-c-masthead--m-light-100--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-masthead--m-light-100__main--BorderBottomColor: var(--pf-global--palette--black-200);
```

Comment:
In the page component, turning a page section white is `.pf-m-light`. Then we have `.pf-m-color-scheme--light-300` in tabs, though that's arguably a different use case.

And we have a drawer issue this sprint (https://github.com/patternfly/patternfly/issues/3813) to enable the grey background on the panel/content. I'd like to see if we can create something consistent. I like this. Maybe just `pf-m-light` and `pf-m-light-200` to align with the page? cc @mcarrano @mceledonia